format/private/format.sh

```diff
@@ -94,9 +94,20 @@ function ls-files {
 
         # TODO: determine which staged changes we should format; avoid formatting unstaged changes
         # TODO: try to format only modified regions of the file (where supported)
-        git ls-files --cached --modified --other --exclude-standard "${patterns[@]}" "${patterns[@]/#/*/}" | {
-          grep -vE "^$(git ls-files --deleted)$" || true;
-        }
+        files=$(git ls-files --cached --modified --other --exclude-standard "${patterns[@]}" "${patterns[@]/#/*/}" | {
+          grep -vE \
+            "^$(git ls-files --deleted)$" \
+          || true;
+        })
+        non_ignored_files=()
+        for file in "${files[@]}"; do
```

Comment:
instead of looping the files and running `git check-attr` N times with one argument each, is it possible to run `git check-attr` once with N arguments?

Reply:
Done